_posts/en/newsletters/2022-10-19-newsletter.md

```diff
@@ -0,0 +1,276 @@
+  consensus rules for taproot don't place any direct limits on the size
+  of witness data.  This was a design element discussed during taproot's
+  development (see [Newsletter #65][news65 tapscript limits]).
+
+    Shortly after the large-witness transaction was confirmed, users
+    began to report that the BTCD full node implementation and LND
+    Lightning Network implementation were failing to provide data from
+    the most recent blocks that were available to Bitcoin Core full
+    nodes.  For BTCD nodes, this meant that transactions which had been
+    recently confirmed were being reported as still unconfirmed.  For
+    LND, it meant that new channels that had recently become ready to
+    use weren't being reported as fully open.
+
+    A developer for both BTCD and LND fixed the problem in BTCD's code,
+    which LND uses as a library, and quickly released new versions for
+    both [LND][lnd 0.15.2-beta] (as mentioned in [last week's
+    newsletter][news221 lnd]) and [BTCD][btcd 0.23.2].  All users of
+    BTCD and LND should upgrade.
+
+    Until a user upgrades their software, they will suffer the
+    lack-of-confirmation problems described above and may also be
+    vulnerable to several attacks.  Some of those attacks require access
+    to significant hash rate (making them expensive and, hopefully,
+    impractical in this case).  Other attacks, particularly those
+    against LND users, require the attacker to risk losing some of their
+    funds in a channel, which is also hopefully a sufficient deterrent.
+    We again recommend upgrade and, further, we recommend that anyone
+    using any Bitcoin software sign up for security announcements from
+    that software's development team.
+
+    After the above disclosures, Loki Verloren [posted][verloren limits]
+    to the Bitcoin-Dev mailing list to suggest that direct limits be
+    added to taproot's witness size.  Greg Sanders [replied][sanders
+    limits] to note that adding limits now would not only increase code
+    complexity but could also lead to people losing their money if they
+    already received bitcoins to a script which requires a large witness
+    to spend.
+
+- **Transaction replacement option:** as reported in Newsletters
+  [#205][news205 rbf] and [#208][news208 rbf], Bitcoin Core merged
+  support for a `mempoolfullrbf` configuration option which defaults to
+  the existing Bitcoin Core behavior of only allowing [RBF
+  replacement][topic rbf] of transactions containing the [BIP125][]
+  signal.  However, if a user sets the new option to true, their node
+  will accept and relay replacement transactions that don't contain the
+  BIP125 signal, provided the replacement transactions follow all of
+  Bitcoin Core's other rules for replacements.
+
+    Dario Sneidermanis [posted][sne rbf] to the Bitcoin-Dev mailing that
+    this new option may be concerning for services which currently accept
```

Comment:
My sense from the ML posts is that "may be concerning" should be "may be harmful". (Just a suggestion.)

Reply:
Keeping as is, this sentence already has a "that".

Going to s/may be concerning/may create problems/